### PR TITLE
Docs configuration

### PR DESCRIPTION
**Pull request by filrak**
- Adds docs about configuration
- Adds link to roadmap
- Minor polishments of i18n docs 
-  Logging docs rewrite so its focusing with Nuxt usage(default one)
- Added scope to Theme docs

### CODE REVIEW

File: packages/core/docs/advanced/internationalization.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Internationalization
 
-If you're building a shop for an international brand you want it being translated to different languages and using different currencies. In this doc you will learn how we're approaching i18n (internationalization) in Vue Storefront.
+If you're building a shop for an international brand you want it being translated to different languages and using different currencies. In this document you will learn how we're approaching internationalization in Vue Storefront and how to configure your application to use
```
Comment (NataliaTepluhina) [inline packages/core/docs/advanced/internationalization.md:3]: **suggestion**: minor grammar fix

```suggestion
If you're building a shop for an international brand you want it to be translated to different languages and using different currencies. In this document, you will learn how we're approaching internationalization in Vue Storefront and how to configure your application to use.
```

File: packages/core/docs/advanced/internationalization.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 ::: tip i18n is not multi-tenancy!
 This document explains only how to make a single shop instance available for multiple countries. If you need to build a system for multiple tenants we suggest creating an instance of Vue Storefront for each tenant and sharing common resources through an NPM package.
 :::
 
-## Default setup
-By default we are using [`nuxt-i18n`](https://i18n.nuxtjs.org/) module for handling both translations and currencies. 
+## How it works by default?
+By default we are using [`nuxt-i18n`](https://i18n.nuxtjs.org/) module for handling both translations and currencies. It is preinstalled in the default theme and configured for English and German translations out of the box.
```
Comment (NataliaTepluhina) [inline packages/core/docs/advanced/internationalization.md:10]: **suggestion**: let's separate headings with an empty row

```suggestion
## How it works by default?

By default, we are using [`nuxt-i18n`](https://i18n.nuxtjs.org/) module for handling both translations and currencies. It is preinstalled in the default theme and configured for English and German translations out of the box. 
```

File: packages/core/docs/guide/configuration.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Configuration
+
+Usually the first thing to do after setting up a fresh Vue Storefront project is configuring it. You have to at least provide the API credentials for your integrations but in most cases you will also want to configure app behavior to match your business requirements.
```
Comment (NataliaTepluhina) [inline packages/core/docs/guide/configuration.md:3]: **suggestion**: some rewording to improve a readability
```suggestion
Usually, the first thing to do after setting up a fresh Vue Storefront project is configuring it. The bare minimum is to provide the API credentials for your integrations but in most cases, you will also want to configure app behavior to match your business requirements.
```

**Review by NataliaTepluhina — COMMENTED** [review]
@filrak great job! I've left a few suggestions here and there for your consideration

File: packages/core/docs/advanced/logging.md [hunk 4]
```diff
@@ -1,106 +1,87 @@
...
 - `debug` - log everything, including debug calls, information, warnings, and errors (all of the logger functions can be called),
 - `info` - log information, warnings, and errors (debug function calling is skipped),
 - `warn` - log warnings and errors (debug and info functions are skipped),
 - `error` - log only errors (debug, warn and info functions are skipped),
 - `none` - don't log anything.
 
+```js
+['@vue-storefront/nuxt', {
+  logger: {
+    verbosity: 'error' // verbosity
```
Comment (Fifciu) [inline packages/core/docs/advanced/logging.md:33]: I think we could get rid of this comment as we have the same on the left

File: packages/core/docs/advanced/logging.md [hunk 5]
```diff
@@ -1,106 +1,87 @@
...
-   error: (message: any, ...args) => console.error('[VSF][error]', message, ...args),
- }
-}
+By default we're printing all the events happening in the app in the console  but you can easily write a new Logger and use a third-party library (like `consola`) or pass the logs to external service like `Sentry.io`
 
-registerLogger(logger, 'error');
-```
-
-## Configuring with nuxt
+To override the default Logger simply pass  a function to the `logger.customLogger` property of `@vue-storefront/nuxt` module. This function returns the logger object and as an argument, you have access to the `verbosity` level.
```
Comment (Fifciu) [inline packages/core/docs/advanced/logging.md:49]: `pass [DOUBLE SPACE] a function to the...` get rid of double space

File: packages/core/docs/advanced/logging.md [hunk 6]
```diff
@@ -1,106 +1,87 @@
...
 }]
 ```
 
-There is a dedicated section called `logger` that's responsible for logger configuration, with possible options:
-
-- `verbosity` - sets the verbosity level
-- `debug`, `info`, `warn`, `error` - you are able to provide each of these functions or even all of them, with keeping default verbosity implementation.
-- `customLogger` - if you provide this property, you are overriding the original logger with your custom one.
+::: details Configuring logger outside Vue Storefront Nuxt module
+If for some reason you can't configure Logger through `@vue-storefront/nuxt`  module you can explicitly use `registerLogger` function:
```
Comment (Fifciu) [inline packages/core/docs/advanced/logging.md:71]: throught @vue-storefront/nuxt [DOUBLE_SPACE] module` - get rid of double space

File: packages/core/docs/guide/configuration.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+We try to use the most common modules from Nuxt Community for these things so for internationalization we are using `nuxt-i18n`, for PWA capabilities `@nuxtjs/pwa` etc. You can find a list of the Nuxt modules used in the default theme [here](theme.html#preinstalled-modules-and-libraries). Each of them is configured in a way that works best for the majority of users.
+
+There are some features and behaviors that are specific to Vue Storefront only yet not specific to a certain integration. You can configure such things through `@vue-storefront/nuxt` module. 
+
+[//]: # (TODO: Add documentation for VSF/NUXT module)
+
+Below you can find its default configuration:
+
+::: danger
+It's unsafe and not recommended to remove  `@vue-storefront/nuxt`  from your project as many integrations and other modules rely on it.
```
Comment (Fifciu) [inline packages/core/docs/guide/configuration.md:20]: to remove [DOUBLE_SPACE] `@vue-storefront/nuxt` [DOUBLE_SPACE] from - get rid of double spaces please

File: packages/core/docs/guide/theme.md [hunk 8]
```diff
@@ -1,6 +1,36 @@
...
+- CMS integration <Badge text="Enterprise" type="info" />
+  - Home Page
+  - Custom CMS Page
+  - Header
+  - Footer
+- Payment integration <Badge text="Enterprise" type="info" />
+
+**Other**
+- Progressive Web App features
+- internationalization
```
Comment (Fifciu) [inline packages/core/docs/guide/theme.md:31]: Let's make it `Internationalization`

**Review by Fifciu — CHANGES_REQUESTED** [review]

File: packages/core/docs/advanced/logging.md (hunk 4 above)
Comment (filrak) [reply packages/core/docs/advanced/logging.md:33]: yup, overlooked :) Thanks!

**Pull request merged**